lib/publishing_api_finder_publisher.rb

```diff
@@ -0,0 +1,80 @@
+require "gds_api/publishing_api"
+require_relative "../app/presenters/finders/finder_content_item_presenter"
+require_relative "../app/presenters/finders/finder_links_presenter"
+require_relative "../app/presenters/finders/finder_signup_content_item_presenter"
+require_relative "../app/presenters/finders/finder_signup_links_presenter"
```

Comment:
These could probably just be `require`s if added to the Rails autoload path, e.g.

https://github.com/alphagov/publishing-api/blob/master/config/application.rb#L29